loopchain/blockchain/transactions/transaction_verifier.py

```diff
@@ -33,18 +62,21 @@ def verify(self, tx: 'Transaction', blockchain=None):
     def verify_loosely(self, tx: 'Transaction', blockchain=None):
         raise NotImplementedError
 
+    @cache_result
```

Comment:
This decorator has to be removed.
Result of this function depends on its parameter.

Reply:
Thank you